340 update joplin community app notebook

ylyangtw opened the pull request:
I refer to SLC community notebook to create a community notebook for Joplin. Then I add one section of retrofit strategy.

We found out that Joplin testbed and Joplin CGE are not using captial shock analysis. Created an issue for this https://github.com/IN-CORE/incore-docs/issues/355

longshuicy commented:
We updated the retrofit fragility mapping key (from retorift strategy 1 to retrofit_1); Hence those RS files I shared with you might not be valid anymore and I deleted them.
I can get you new ones later this week.

ywkim312 commented:
Changelog?

ywkim312 commented on notebooks/joplin_community_app.ipynb:
```diff
@@ -0,0 +1,4407 @@
+   ],
+   "source": [
+    "# Create IN-CORE client and connection to services\n",
+    "client.clear_cache\n",
```
this line should go after the next line.

ywkim312 commented:
I got following error in 8.2 Building Damage Analysis for 3 Retrofit Strategies
```
ERROR - http_util.py:return_http_response() - A HTTPError has occurred 
HTTP Status code: 404
Error Message: Could not find the dataset 65f861fbbc6b5959ba479894
```

ylyangtw commented:
I have updated the 3 retrofit strategies in prod and in notebook, and added changelog. Also added additional steps for printing functionality probabilities and samples, but keep them in comment so users can choose whether they want to see those details.

ywkim312 reviewed (approved):
Notebook ran till the end successfully. Saw many red colored warning text that might confuses the user for error but it is not the problem of only this one.
